libc/cmake/modules/LLVMLibCFlagRules.cmake

```diff
@@ -134,7 +134,7 @@ set(FMA_OPT_FLAG "FMA_OPT")
 set(ROUND_OPT_FLAG "ROUND_OPT")
 # This flag will define macros that gated away vectorization code such that
 # one can always test the fallback generic code path.
-set(PREFER_GENERIC_FLAG "PREFER_GENERIC")
+set(EXPLICIT_SIMD_OPT_FLAG "EXPLICIT_SIMD_OPT")
```

Comment:
Currently `EXPLICIT_SIMD_OPT` is only available / make sense for x86_64, so you should add
```
if(NOT LIBC_TARGET_ARCHITECTURE_IS_X86)
  set(SKIP_FLAG_EXPANSION_EXPLICIT_SIMD_OPT TRUE)
endif()
```
to ignore and not set this flag on non-x86 targets.

Reply:
Sounds good.